### PR TITLE
RUN-809: Moving packaging-core to core repo

### PR DESCRIPTION
**Pull request by carlosrfranco**
Moves content of packaging-core repo to rundeck repo

### CODE REVIEW

**Comment by carlosrfranco** [comment]
@Mergifyio rebase

**Comment by carlosrfranco** [comment]
@Mergifyio rebase

**Comment by chrismcg14** [comment]
This is looking good. A couple of minor items:

1. the packaging submodule seems to have gotten committed
2. Restore proper version before merge

File: packaging/scripts/circle-build.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                -PpackagePrefix="rundeck-" \
+                -PpackageType=$PACKAGE \
+                -PpackageOrg=rundeck \
+                -PpackageRevision=1 \
+                publish
+        done
+    )
+}
+
+publish_pro() {
```
Comment (chrismcg14) [inline packaging/scripts/circle-build.sh:87]: these pro methods can be removed since another build.sh file was added in pro

**Review by chrismcg14 — CHANGES_REQUESTED** [review]
v

**Comment by gschueler** [comment]
fyi: a change to packaging template will need to be incorporated here as well https://github.com/rundeck/packaging/pull/36

File: .circleci/config.yml [hunk 2]
```diff
@@ -569,20 +755,23 @@ workflows:
...
           filters:
-            branches: {ignore: /.*/}
-            tags: {only: /^v.*$/}
+            branches: { ignore: /.*/ }
+            tags: { only: /^v.*$/ }
+      - packaging_test:
+          name: Test Packaging
+          requires: *stage-build
+          filters:
+            branches: { only: /^RUN-809.*/ }
```
Comment (gschueler) [inline .circleci/config.yml:768]: let's enable this for all branches, then we can merge

File: docker/debinstall/entry.sh [hunk 3]
```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 . /rd-util.sh
 
-test -f "$HOME"/rundeck/packaging/debdist/rundeck*.deb || {
-	echo "debian not found at $HOME/rundeck/packaging/debdist/rundeck*.deb"
+test -f "$HOME"/packaging/packaging/debdist/rundeck*.deb || {
```
[the remark this answers is not in the document]
Comment (gschueler) [reply docker/debinstall/entry.sh:4]: nm, we can do this in a different pr

**Review by gschueler — APPROVED** [review]

**Pull request merged**